docs/atlascli/command/atlas-dbusers-update.txt

```diff
@@ -85,6 +85,10 @@ Options
      - string
      - false
      - Username for authenticating to MongoDB.
+   * - --x509Type
+     - string
+     - false
+     - X.509 method by which the provided username is authenticated.  Valid values are NONE, MANAGED, or CUSTOMER. If you set this to MANAGED the user authenticates with an Atlas-managed X.509 certificate. If you set this to CUSTOMER, the user authenticates with a self-managed X.509 certificate. If you set this to MANAGED or CUSTOMER, you can't set --awsIAMType or --ldapType to any value other than NONE. This value defaults to "NONE".
```

Comment:
```suggestion
     - X.509 method for authenticating the specified username.  Valid values include:
     
       .. list-table::
          :header-rows: 1
          
          * - Value
            - Description
            
          * - NONE
            - Default.
           
          * - MANAGED
            - The user authenticates with an Atlas-managed X.509 certificate.
           
          * - CUSTOMER
            -  The user authenticates with a self-managed X.509 certificate.
           
       If you set this to MANAGED or CUSTOMER, you can't set --awsIAMType
       or --ldapType to any value other than NONE.
```

Reply:
This is the printed output from the CLI rather than API docs, so I think the above format would be misaligned with the existing format

Below is how it appears for `atlas dbuser update -h` with the PR's current changes

```

Usage:
  atlas dbusers update <username> [flags]

Examples:
  # Update roles for a database user named myUser for the project with the ID 5e2211c17a3e5a48f5497de3:
  mongocli dbuser update myUser --role readWriteAnyDatabase --projectId 5e2211c17a3e5a48f5497de3

  # Update scopes for a database user named myUser for the project with the ID 5e2211c17a3e5a48f5497de3:
  mongocli dbuser update myUser --scope resourceName:resourceType --projectId 5e2211c17a3e5a48f5497de3

Flags:
      --authDB string      Authentication database name. If the user authenticates with AWS IAM, x.509, or LDAP, this value should be $external. If the user authenticates with SCRAM-SHA, this value should be admin.
  -h, --help               help for update
  -o, --output string      Output format. Valid values are json, json-path, go-template, or go-template-file. To see full output, use the -o json option.
  -p, --password string    Password for the database user.
      --projectId string   Hexadecimal string that identifies the project to use. This option overrides the settings in the configuration file or environment variable.
      --role strings       User's roles and the databases or collections on which the roles apply. Passing this flag replaces preexisting data.
      --scope strings      Array of clusters and Atlas Data Lakes that this user has access to. Passing this flag replaces preexisting data.
  -u, --username string    Username for authenticating to MongoDB.
      --x509Type string    X.509 method by which the provided username is authenticated.  Valid values are NONE, MANAGED, or CUSTOMER. If you set this to MANAGED the user authenticates with an Atlas-managed X.509 certificate. If you set this to CUSTOMER, the user authenticates with a self-managed X.509 certificate. If you set this to MANAGED or CUSTOMER, you can't set --awsIAMType or --ldapType to any value other than NONE. (default "NONE")

Global Flags:
  -P, --profile string   Name of the profile to use from your configuration file. To learn about profiles for the Atlas CLI, see https://dochub.mongodb.org/core/atlas-cli-save-connection-settings.


```

Reply:
Here is the existing set of usage strings in case we wanted to make amendments/align the format/make amendments:
https://github.com/mongodb/mongodb-atlas-cli/blob/master/internal/usage/usage.go

Existing x509Type string is here https://github.com/mongodb/mongodb-atlas-cli/blob/e576fbb3ce62ed9408b40706e19fcaae4950b84c/internal/usage/usage.go#L342

Reply:
Do you think maybe a language amendment would suffice?

```
X509Type                                  = "X.509 method for authenticating the specified username.  Valid values include NONE, MANAGED, and CUSTOMER. If you set this to MANAGED or CUSTOMER, you can't set --awsIAMType or --ldapType to any value other than NONE." 
```

Reply:
The language amendment suffices! Thanks.